File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ExpandableGroupBox/Functionality/TextImagePage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.ExpandableGroupBox.Functionality
{
    public partial class TextImagePage : UserControl
    {
        public TextImagePage()
        {
            InitializeComponent();

        }
        private void mobjIsSpreadCheck_CheckedChanged(object sender, EventArgs e)
        {
            //Change ExpandableGroupBox.IsTextImageSpread property
            mobjExpandableGroupBox.IsTextImageSpread = mobjIsSpreadCheck.Checked;
        }

        /// <summary>
        /// Handles the CheckedChanged event of the mobjImageBeforeRB control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjImageBeforeRB_CheckedChanged(object sender, EventArgs e)
        {
            //Change ExpandableGroupBox.TextImageRelation property
            if (mobjImageBeforeRB.Checked)
                mobjExpandableGroupBox.TextImageRelation = Gizmox.WebGUI.Forms.ExpandableGroupBox.HorizontalTextImageRelation.ImageBeforeText;
        }

        /// <summary>
        /// Handles the CheckedChanged event of the mobjTextBeforeRB control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjTextBeforeRB_CheckedChanged(object sender, EventArgs e)
        {
            //Change ExpandableGroupBox.TextImageRelation property
            if (mobjTextBeforeRB.Checked)
                mobjExpandableGroupBox.TextImageRelation = Gizmox.WebGUI.Forms.ExpandableGroupBox.HorizontalTextImageRelation.TextBeforeImage;
        }


    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ExpandableGroupBox/Functionality/TextImagePage.vb
Namespace CompanionKit.Controls.ExpandableGroupBox.Functionality

	Public Class TextImagePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjIsSpreadCheck control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjIsSpreadCheck_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjIsSpreadCheck.CheckedChanged
            'Change ExpandableGroupBox.IsTextImageSpread property
            mobjExpandableGroupBox.IsTextImageSpread = mobjIsSpreadCheck.Checked
        End Sub

       
        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjImageBeforeRB control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjImageBeforeRB_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjImageBeforeRB.CheckedChanged
            'Change ExpandableGroupBox.TextImageRelation property
            If mobjImageBeforeRB.Checked Then
                mobjExpandableGroupBox.TextImageRelation = Gizmox.WebGUI.Forms.ExpandableGroupBox.HorizontalTextImageRelation.ImageBeforeText
            End If
        End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjTextBeforeRB control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjTextBeforeRB_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjTextBeforeRB.CheckedChanged
            'Change ExpandableGroupBox.TextImageRelation property
            If mobjTextBeforeRB.Checked Then
                mobjExpandableGroupBox.TextImageRelation = Gizmox.WebGUI.Forms.ExpandableGroupBox.HorizontalTextImageRelation.TextBeforeImage

            End If
        End Sub
    End Class

End Namespace